auth0/src/main/java/com/auth0/android/request/internal/BaseRequest.java

```diff
@@ -136,6 +138,11 @@ protected U parseUnsuccessfulResponse(Response response) {
         } catch (IOException e) {
             final Auth0Exception auth0Exception = new Auth0Exception("Error parsing the server response", e);
             return errorBuilder.from("Request to " + url.toString() + " failed", auth0Exception);
+        } finally {
+            try {
+                body.close();
+            } catch (Exception ignored) {
```

Comment:
Should we only catch `IOException` instead all of them so we know when we fail to proper manage the errors?